PackageBuilder/MultiPackageListing.cs

```diff
@@ -138,7 +138,7 @@ private async Task<List<IVRCPackage>> GetPackagesFromGitHubRepo(string ownerSlas
                         var discoveredPackages = await GetPackagesFromGitHubRepo(ownerSlashName);
                         if (discoveredPackages != null && discoveredPackages.Count > 0)
                         {
-                            packages.AddRange(discoveredPackages);
+                            packages.AddRange(discoveredPackages.ConvertAll(p => (VRCPackageManifest) p));
```

Comment:
I didn't know about `ConvertAll()` and I think this is the second time I've seen you use it now, pretty handy.

Reply:
Yeah, it was a Roslyn analyzer plugin I had that recommended using it instead.
When the types have built-in conversion support you don't even need to pass a lambda.